src/nnue/nnue_architecture.h

```diff
@@ -46,7 +46,7 @@ struct Network
   static constexpr int FC_1_OUTPUTS = 32;
 
   Layers::AffineTransform<TransformedFeatureDimensions, FC_0_OUTPUTS + 1> fc_0;
-  Layers::ClippedReLU<FC_0_OUTPUTS> ac_0;
+  Layers::ClippedReLU<FC_0_OUTPUTS + 1> ac_0;
```

Comment:
@Sopel97 can you have a look at these changes as well? I'm not sure I understand why this is equivalent here.

Reply:
it allocates a larger buffer and considers the input to be larger (and consequently output). If the buffered i zeroed it is okay and may even increase performance since SIMD implementation kicks in after some threshold size.

Reply:
OK, makes sense, and could explain why it looks slightly positive in SPRT tests.

Reply:
We only need to clip 15 elements as shown here https://user-images.githubusercontent.com/8037982/153455685-cbe3a038-e158-4481-844d-9d5fccf5c33a.png in the Layer Stack diagram.  Adding a 16th unused element just for padding allows the SIMD portion of the code path in ClippedReLU::propagate() in clipped_relu.h line 83 to be used instead of the generic non SIMD loop on line 169.